### PR TITLE
[Telemetry] Include the buckets in the typing event

### PR DESCRIPTION
**Pull request by iainx**
Fixes VSTS #639164

### CODE REVIEW

**Comment by iainx** [comment]
Once https://github.com/mono/monodevelop/pull/5093 is merged this will need changed to not submit timings and counts as strings

**Comment by iainx** [comment]
@mrward I haven't updated the one note yet, there's a lot needs to go into it

File: main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor/Gui/TextEditorKeyPressTimings.cs [hunk 1]
```diff
@@ -53,6 +53,15 @@ class TextEditorKeyPressTimings
 		int activeCountIndex = 0;
 		int droppedEvents = 0;
 
+		const int numberOfBuckets = 9;
+		readonly int[] buckets = new int[numberOfBuckets];
+
+		// One less than buckets because the last bucket is everything else.
+		// This number is the max time a keystroke can take to be placed into this bucket
+		int[] bucketUpperLimit = {
```
Comment (slluis) [inline main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor/Gui/TextEditorKeyPressTimings.cs]: This should be static.

Comment (Therzok) [reply main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor/Gui/TextEditorKeyPressTimings.cs]: `static readonly ImmutableArray<int>`?

**Comment by iainx** [comment]
monojenkins backport release-7.6

**Comment by slluis** [comment]
@monojenkins backport release-7.6

**Pull request merged**